addOns/pscanrules/src/main/javahelp/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html

```diff
@@ -287,7 +287,13 @@ <H2>PII Disclosure</H2>
 At MEDIUM and HIGH threshold it attempts to use three characters of context on each side of potential matches to exclude matches within decimal like content.
 At LOW threshold, alerts will be raised for such matches.
 <p>
-PDFs are only evaluated at LOW threshold.
+At DEFAULT, MEDIUM AND HIGH Threshold, the following content types are evaluated:
```

Comment:
Remove default. The user can set default to whatever they want. Also 'and' doesn't need caps.